src/components/Pagination.vue

```diff
@@ -84,10 +84,17 @@ export default {
 
       return pages;
     },
+    totalResults() {
+      return this.searchStore.totalResults;
+    },
   },
   methods: {
     goToPage(page) {
-      this.searchStore.page = Math.min(this.totalPages, page);
+      let p = page;
```

Comment:
Moreover, pagination with Algolia is zero-based (see https://www.algolia.com/doc/guides/searching/pagination/#response-information) so shouldn't you check for `0` instead of `1`?